src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -210,6 +210,10 @@ public boolean isActive(long deviceId) {
     return isActive(deviceControllerPtr, deviceId);
   }
 
+  public void shutdownSubscriptions(long devicePtr) {
```

Comment:
Probably worth commenting that it shuts down all subscriptions for the device, just to be clear.

Reply:
Done